pkgs/applications/office/appflowy/default.nix

```diff
@@ -47,6 +47,10 @@ stdenv.mkDerivation rec {
     # Copy archive contents to the outpout directory
     cp -r ./* $out/opt/
 
+    # Copy icon
+    install -Dm444 data/flutter_assets/assets/images/flowy_logo.svg $out/share/pixmaps/appflowy.svg
+    install -Dm444 data/flutter_assets/assets/images/flowy_logo.svg $out/share/icons/hicolor/scalable/apps/appflowy.svg
```

Comment:
Sorry, I should have specified that was an alternative. You should only need one of these 😅

Reply:
Sorry, I'm tired :/

Reply:
Aren't we all 🫠